db/migrate/20181128174938_add_lineage_version_to_alignment_config.rb

```diff
@@ -0,0 +1,21 @@
+class AddLineageVersionToAlignmentConfig < ActiveRecord::Migration[5.1]
+  def up
+    # Max 32767
+    add_column :alignment_configs, :lineage_version, :integer, limit: 2
+    AlignmentConfig.where(name: "2018-02-15").update(lineage_version: 1)
+    AlignmentConfig.where(name: "2018-04-01").update(lineage_version: 2)
+
+    # It will go from alignment lineage_version to "lineage records with a [version_start, version_end] range that includes that number."
```

Comment:
Can you add a comment like:
"We use the lineage_version to figure out which lineage records to use in our pipeline run. Lineage records are highly duplicated between versions, so instead of storing all of them, we store a [version_start, version_end] range with each lineage record."

Mentioning that the lineage records are duplicated is really helpful for understanding the purpose of this I think.